includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1500,12 +1526,12 @@ protected function get_metadata_from_order( $order, $payment_type ) {
 
 			foreach ( $subscriptions as $subscription ) {
 				if ( WC_Payments_Subscription_Service::is_wcpay_subscription( $subscription ) ) {
-					$metadata['payment_context'] = 'wcpay_subscription';
+					$metadata['payment_context']      = 'wcpay_subscription';
```

Comment:
Notice how `is_wcpay_subscription` is called here, and the whole section is only entered if `! $this->is_subscriptions_plugin_active()`. We need a separate block for standard subscriptions, one where `'recurring' === (string) $payment_type && $this->is_subscriptions_plugin_active()`.